### PR TITLE
Display fixes for FL wait time section

### PR DESCRIPTION
**Pull request by patrickvinograd**
- Removes Primary Care from being duplicated under Specialty care subhead
- Allows established patient wait times to show by fixing key lookup (`existing` -> `established`)
- Avoids showing "See more" unless there are more elements to show.
- Alphabetizes specialty care categories for consistent ordering.

### CODE REVIEW

File: src/js/facility-locator/components/AppointmentInfo.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import pluralize from 'pluralize';
-import { startCase } from 'lodash';
+import { pull, startCase } from 'lodash';
```
Comment (bshyong) [inline src/js/facility-locator/components/AppointmentInfo.jsx]: nice 🍸

**Review by bshyong — APPROVED** [review]
LGTM thanks for fixing!

File: src/js/facility-locator/components/AppointmentInfo.jsx [hunk 2]
```diff
@@ -26,7 +26,9 @@ export default class AppointmentInfo extends Component {
 
     const healthAccessAttrs = facility.attributes.access.health;
     const specialtyKeys = healthAccessAttrs && Object.keys(healthAccessAttrs);
-    delete specialtyKeys.primaryCare;
+    pull(specialtyKeys, 'primaryCare');
+    pull(specialtyKeys, 'effectiveDate');
```
Comment (U-DON) [inline src/js/facility-locator/components/AppointmentInfo.jsx]: Looks like `pull` can pass multiple arguments for elements to remove. https://lodash.com/docs/4.17.4#pull

```
pull(specialtyKeys, 'primaryCare', 'effectiveDate');
```

@bshyong: There's a merge conflict with these changes on one of your PRs if you want to make the change there.

**Review by U-DON — COMMENTED** [review]
LGTM.

File: src/js/facility-locator/components/AppointmentInfo.jsx (hunk 2 above)
Comment (bshyong) [inline src/js/facility-locator/components/AppointmentInfo.jsx]: Thanks will make those changes in https://github.com/department-of-veterans-affairs/vets-website/pull/5749

**Pull request merged**